docs/cli.rst

```diff
@@ -48,6 +51,12 @@ Or with a filename::
     export FLASK_APP=/path/to/hello.py
     flask run
 
+If you use Flask 0.13 or later you can also set the environment variables
+in a ``.env`` file.  For instance like this::
+
+    FLASK_APP=/path/to/hello.py
+    FLASK_DEBUG=1
+
```

Comment:
Because that's how a .env file is used in a lot of other frameworks. That's the first thing I think about with `.env` files. I actually have implemented a `.env` importer for my own Flask projects as well, specifically for secrets. It's a pretty standard practice, I think.